File: v2/csharp/src/main/csharp/io/swagger/Model/Sourceresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Sourceresponse {
    

    
    public Source Source { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Sourceresponse {\n");
      
      sb.Append("  Source: ").Append(Source).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Sourceresponse.java
package vestorly.client.model;

import vestorly.client.model.Source;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Sourceresponse  {
  
  private Source source = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("source")
  public Source getSource() {
    return source;
  }
  public void setSource(Source source) {
    this.source = source;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Sourceresponse {\n");
    
    sb.append("  source: ").append(source).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
